iptables/iptables.go

```diff
@@ -45,6 +45,7 @@ var (
 	iptablesPath  string
 	supportsXlock = false
 	supportsCOpt  = false
+	xLockWaitMsg  = "Another app is currently holding the xtables lock; waiting"
```

Comment:
do we need the `; waiting` to be part of the check ? It seems that has more probability of changing ?
Also, there is no other way around for this ?